Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md

```diff
@@ -0,0 +1,71 @@
+# The Umbraco Heartcore Portal
+
```

Comment:
Should we add some intro text here, explaining what kind of information can be found in the article?